From: "Torne (Richard Coles)" <torne@google.com>
Date: Fri, 22 Jul 2022 21:59:58 +0000
Subject: Suppress lint in DarkModeHelper for T SDK.

Linting against the T SDK results in a new lint in DarkModeHelper;
suppress it as we have no alternative here.

Bug: 1339376
Change-Id: Id0785fb3ea10e9de4d8fd1c4c13ffa5e2fe6dfc0
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/3780314
Auto-Submit: Richard Coles <torne@chromium.org>
Reviewed-by: Michael Bai <michaelbai@chromium.org>
Commit-Queue: Richard Coles <torne@chromium.org>
Commit-Queue: Michael Bai <michaelbai@chromium.org>
Cr-Commit-Position: refs/heads/main@{#1027430}
---
 .../java/src/org/chromium/android_webview/DarkModeHelper.java   | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/android_webview/java/src/org/chromium/android_webview/DarkModeHelper.java b/android_webview/java/src/org/chromium/android_webview/DarkModeHelper.java
--- a/android_webview/java/src/org/chromium/android_webview/DarkModeHelper.java
+++ b/android_webview/java/src/org/chromium/android_webview/DarkModeHelper.java
@@ -64,6 +64,8 @@ public class DarkModeHelper {
         }
     }
 
+    // must use getIdentifier to access resources from another app
+    @SuppressWarnings("DiscouragedApi")
     @LightTheme
     public static int getLightTheme(Context context) {
         if (sLightThemeForTesting != null) return sLightThemeForTesting;
-- 
2.37.3

